api/envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.proto

```diff
@@ -19,22 +19,40 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#extension: envoy.filters.http.aws_request_signing]
 
 // Top level configuration for the AWS request signing filter.
-// [#next-free-field: 6]
+// [#next-free-field: 7]
 message AwsRequestSigning {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.http.aws_request_signing.v2alpha.AwsRequestSigning";
 
+  enum SigningAlgorithm {
+    // Use SigV4 for signing
+    AWS_SIGV4 = 0;
+
+    // Use SigV4A for signing
+    AWS_SIGV4A = 1;
+  }
+
   // The `service namespace
   // <https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-aws-service-namespaces>`_
   // of the HTTP endpoint.
   //
   // Example: s3
   string service_name = 1 [(validate.rules).string = {min_len: 1}];
 
-  // The `region <https://docs.aws.amazon.com/general/latest/gr/rande.html>`_ hosting the HTTP
-  // endpoint.
+  // When signing_algorithm = aws_sigv4 the region is a standard AWS `region <https://docs.aws.amazon.com/general/latest/gr/rande.html>`_ string for the service
+  // hosting the HTTP endpoint.
```

Comment:
```
When ``signing_algorithm`` is set to ``AWS_SIGV4`` or is not set explicitly, the region is .....
```

Reply:
updated to caps formatting for all references of this enum